server/src/test/java/org/elasticsearch/index/engine/InternalEngineTests.java

```diff
@@ -4769,7 +4775,7 @@ public void testSeqNoGenerator() throws IOException {
                     VersionType.INTERNAL,
                     Engine.Operation.Origin.PRIMARY,
                     System.currentTimeMillis(),
```

Comment:
Kind of unrelated, but given next comment, I think this `startTime` should also change into:

```suggestion
                     System.nanoTime(),
```